book/src/user/regtest.md

```diff
@@ -0,0 +1,61 @@
+# Regtest with Zebra
+
+The Regtest network in Zebra enables testing of custom functionalities in a private testnet environment with configurable network upgrade activation heights. It allows for starting an isolated node which won't connect to any peers and currently allows for committing blocks without validating their Proof of Work (in the future, it may use a very low target difficulty and easier Equihash parameters instead of skipping Proof of Work validation altogether).
+
+Zebra always activates the Canopy network upgrade at block height 1 due to limitations on its block construction.
+
+In order to use Regtest, Zebra must be configured to run on the Regtest network. The `[mining]` section is also necessary for mining blocks, and the `[rpc]` section is necessary for using the `send_raw_transaction` RPC method to mine non-coinbase transactions onto the chain.
+
+Relevant parts of the configuration file:
+
+```toml
+[mining]
+miner_address = 't27eWDgjFYJGVXmzrXeVjnb5J3uXDM9xH9v'
+    
+[network]
+network = "Regtest"
+
+// This section may be omitted when testing only Canopy
+[network.testnet_parameters.activation_heights]
+// Configured activation heights must be greater than or equal to 1, 
+// block height 0 is reserved for the Genesis network upgrade in Zebra
+NU5 = 1 
+
+// This section may be omitted if a persistent Regtest chain state is desired
+[state]
+ephemeral = true
+
+// This section may be omitted if it's not necessary to send transactions to Zebra's mempool
+[rpc]
+listen_addr = "0.0.0.0:18232"
+```
+
```

Comment:
I think we need to show zebra start command and zebra running in regtest mode, there should be some log that indicates this is the case.

Reply:
The network name is in every log message, I added an example of what to look for in https://github.com/ZcashFoundation/zebra/pull/8526/commits/0b4991a2d4fd4f4b49c97259d714badeea4ef8cc. I'll add a suggestion for using `cargo run` instead of `cargo build`.